### ISSUE TITLE
sprite.draw() not drawing

### ISSUE THREAD

**Issue by MrGallo** [pvcraven/arcade#575]
Arcade successfully draws a sprite in a spritelist, but not directly using `sprite.draw()`. Commit b66e0afdb231664c9795ce9e9a027b5b46354c18

```python
import arcade

SCREEN_WIDTH = 800
SCREEN_HEIGHT = 600
SCREEN_TITLE = "Resources"


class MyGame(arcade.Window):
    def __init__(self, width, height, title):
        super().__init__(width, height, title)
        arcade.set_background_color(arcade.color.AMAZON)

        self.sprite = arcade.Sprite(':resources:/images/animated_characters/male_adventurer/maleAdventurer_idle.png', center_x=50, center_y=100)

        self.sprite_list = arcade.SpriteList()
        self.sprite_list.append(self.sprite)

    def on_draw(self):
        arcade.start_render()
        self.sprite.draw()  # doesn't work
        # self.sprite_list.draw()  # works

    def update(self, delta_time):
        self.sprite.update()


def main():
    """ Main method """
    game = MyGame(SCREEN_WIDTH, SCREEN_HEIGHT, SCREEN_TITLE)
    arcade.run()


if __name__ == "__main__":
    main()


```

**Comment by MrGallo** [comment]
works now

**Comment by MrGallo** [comment]
Resurrection.

**Comment by pvcraven** [comment]
Weird, works if scale is anything but 1.0

**Comment by pvcraven** [comment]
Fixed in 2.3.5.
